tests/vpp.cpp

```diff
@@ -29,6 +29,8 @@
 
 using namespace YamiMediaCodec;
 
+#define COLOR_BALANCE_DEFAULT -1
```

Comment:
Most ok to me.But this should be COLORBALANCE_LEVEL_NONE. currently it's same value -1, but in future we may change COLORBALANCE_LEVEL_NONE to other value, so use COLORBALANCE_LEVEL_NONE for default is better